news/_posts/core-monthly/2022-12-05-core-monthly-2022-11-01-2022-11-30.md

```diff
@@ -0,0 +1,439 @@
+* [#160](https://github.com/PrestaShop/ps_linklist/pull/160): Configure typescript properly to fix the build. Thank you, [@NeOMakinG](https://github.com/NeOMakinG)
+* [#159](https://github.com/PrestaShop/ps_linklist/pull/159): Bump minimatch from 3.0.4 to 3.1.2 in /views. Thank you, [@dependabot](https://github.com/dependabot)
+* [#158](https://github.com/PrestaShop/ps_linklist/pull/158): Bump glob-parent from 3.1.0 to 5.1.2 in /views. Thank you, [@dependabot](https://github.com/dependabot)
+* [#157](https://github.com/PrestaShop/ps_linklist/pull/157): Bump loader-utils from 1.4.0 to 1.4.1 in /views. Thank you, [@dependabot](https://github.com/dependabot)
+### Stylelint configuration
+* [#55](https://github.com/PrestaShop/stylelint-config/pull/55): Bump stylelint from 14.14.1 to 14.15.0. Thank you, [@dependabot](https://github.com/dependabot)
+* [#53](https://github.com/PrestaShop/stylelint-config/pull/53): Bump stylelint from 14.14.0 to 14.14.1. Thank you, [@dependabot](https://github.com/dependabot)
+### Traces
+* [#37](https://github.com/PrestaShop/traces/pull/37): Remove archived repositories. Thank you, [@Progi1984](https://github.com/Progi1984)
+### OnBoarding module
+* [#150](https://github.com/PrestaShop/welcome/pull/150): Bump css-loader from 6.7.1 to 6.7.2. Thank you, [@dependabot](https://github.com/dependabot)
+* [#149](https://github.com/PrestaShop/welcome/pull/149): Bump minimatch from 3.0.4 to 3.0.8. Thank you, [@dependabot](https://github.com/dependabot)
+
+
+<hr />
+
+Thank you to the contributors whose pull requests were merged since the last Core Monthly Report: @FabienPapet, @HanaRebaiQA, @Hlavtox, @NeOMakinG, @PrestaEdit, @PrestaSafe, @Prestaworks, @Progi1984, @PululuK, @Quetzacoalt91, @Triloworld, @angelo983, @atomiix, @boubkerbribri, @carmas123, @cfarhani06, @davidglezz, @dependabot, @elboletaire, @emmanuelgautier, @eternoendless, @florine2623, @hendaghanmi, @idnovate, @javdome, @jf-viguier, @jolelievre, @khouloudbelguith, @korbi551, @kpodemski, @lartist, @leemyongpakvn, @lmeyer1, @margud, @marsaldev, @matks, @matthieu-rolland, @mflasquin, @micka-fdz, @micreas, @mparvazi, @nesrineabdmouleh, @nicosomb, @okom3pom, @panariga, @rmilecki, @sowbiba, @thomasnares, @webeshop, @yanmakouf, @zuk3975!
+
+Thank you to the contributors whose PRs haven't been merged yet! And of course, a big thank you to all those who contribute with issues and comments [on GitHub](https://github.com/PrestaShop/PrestaShop)!
+
+If you want to contribute to PrestaShop with code, please read these pages first:
+
+ * [Contributing code to PrestaShop](https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/)
+ * [Coding standards](https://devdocs.prestashop-project.org/8/development/coding-standards/)
+
+...and if you do not know how to fix an issue but wish to report it, please read this: [How to use GitHub to report an issue](https://devdocs.prestashop-project.org/8/contribute/contribute-reporting-issues/). Thank you!
```

Comment:
This part could be updated with https://www.prestashop-project.org/get-involved/